tests/integration/components/sticky-element-test.js

```diff
@@ -346,6 +346,41 @@ module('Integration | Component | sticky element', function(hooks) {
       assert.dom('#debug').hasText(debug, debug);
     });
 
+    test(`Is resizable | Size: ${testCase.size}, offView: ${testCase.offView}, stick to bottom: ${testCase.stickToBottom === false ? 'false' : 'true'}, scroll position: ${testCase.scrollPosition}`, async function(assert) {
```

Comment:
This is within the `testCases.forEach()` loop. I am not sure we really need this? 
The loop is more for all the permutations regarding to positioning, for the resize tests it's maybe enough to put it outside of the loop, so tests run a bit faster?